matlab/+stimulus/+core/Screen.m

```diff
@@ -44,9 +44,10 @@ function open(self)
                 AssertOpenGL
                 sca
                 % pix screen with the largest screen number
-                screen = max(Screen('Screens'));
-                [self.win, self.rect] = Screen('OpenWindow',screen,127,[],[],[],[],[], ...
-                    mor(kPsychNeedFastBackingStore,kPsychNeed16BPCFloat));
+                screen = 0; % max(Screen('Screens'));
+                [self.win, self.rect] = Screen('OpenWindow', screen, 127, ...
+                    [0, 0, 800, 600] ,[],[],[],[], ...
+                    mor(kPsychNeedFastBackingStore, kPsychNeed16BPCFloat));
```

Comment:
In [PR 4](https://github.com/datajoint/element-visual-stimulus/pull/4/files#diff-68a3403c6afa5954d331f6319302f142147b3ca9ae9cbe12b06935e65e65514bR75) I added a separate function for opening a small window vs. full screen. Does this offer that option, or is the screen size set here at 800x600? If this is intended for demo, I would set the y corner at 22 to adjust the height of the menu par for Mac users
```suggestion
                [self.win, self.rect] = Screen('OpenWindow', screen, 127, ...
                    [0, 22, 800, 600] ,[],[],[],[], ...
                    mor(kPsychNeedFastBackingStore, kPsychNeed16BPCFloat));
```

Reply:
yes, we should parameterize this. Typically, the display is used in full screen, so the size should just be omitted.